src/NeoSharp.BinarySerialization/BinarySerializer.cs

```diff
@@ -36,17 +42,13 @@ static BinarySerializer()
         /// <param name="asm">Assembly</param>
         public static void CacheTypesOf(Assembly asm)
         {
-            foreach (var t in asm.GetTypes())
+            foreach (var t in asm.GetTypes().Where(t => typeof(TypeConverter).IsAssignableFrom(t)))
+                InternalCacheTypeConvertersOf(t);
+
+            foreach (var t in asm.GetTypes().Where(t => typeof(TypeConverter).IsAssignableFrom(t) == false))
                 InternalCacheTypesOf(t);
```

Comment:
I think that is not the same, maybe the type converter don't produce the same result as neo (old version) want for some types

Reply:
I am not sure which types you mean but feel free to change it. My goal was to make it works for UInt160/256 quickly to pass tests.